### ISSUE TITLE
Yelp dataset

### ISSUE THREAD

**Issue by MARAM202** [luojijiaren/YelpRecommendation#2]
Hello community, I would like to run this source code https://github.com/luojijiaren/YelpRecommendation but the link shared for the yelp dataset is not working, can any one help please?

**Comment by luojijiaren** [comment]
Hey, the original dataset is not existed anymore. you can search yelp dataset in Kaggle.

**Comment by MARAM202** [comment]
First of all thanks for your respond, and can I use the source code with
movielens dataset?
Since i couln't find the dataset, i can't run the code can you help me
through this please

Le jeu. 9 juil. 2020 à 15:21, Fangling Zhang <notifications@github.com> a
écrit :

**Comment by luojijiaren** [comment]
Yes, you can use the source code if you can credit my github link when you public the code, I'll appreciate it. Thanks

**Comment by MARAM202** [comment]
Yes that's for sure . Otherwise, do you know any works ( code sources,
scientific paper that propose a combination between RBM and XGBooster for
recommendrr systems)?

Le jeu. 9 juil. 2020 à 15:29, Fangling Zhang <notifications@github.com> a
écrit :

**Comment by MARAM202** [comment]
Hello, sorry if I'm texting you again, but I wanted to ask if you can give
a resume about the code shared ( steps) because there is no enough comments
to better understand the code.

I'll be so grateful for your help

Le jeu. 9 juil. 2020 à 16:56, belkhadir imane <belk.imanee@gmail.com> a
écrit :
